src/encoded/__init__.py

```diff
@@ -141,13 +141,13 @@ def robots_txt(request):
     config.add_view(robots_txt, route_name='robots.txt-conditional')
 
 
-def load_workbook(app, workbook_filename, docsdir):
-    environ = {
-        'HTTP_ACCEPT': 'application/json',
-        'REMOTE_USER': 'IMPORT',
-    }
-    testapp = VirtualApp(app, environ)
-    load_all(testapp, workbook_filename, docsdir)
+# def load_workbook(app, workbook_filename, docsdir):
```

Comment:
In case this function isn't just totally on it's way out...

If this is useful, maybe it should move to either util.py or to loadxl.py itself.

Also, speaking of utility functions, there is a function that makes a testapp of the appropriate kind, and people shouldn't be composing them raw like this in my view.